src/qt/walletmodel.cpp

```diff
@@ -247,7 +252,7 @@ WalletModel::SendCoinsReturn WalletModel::sendCoins(WalletModelTransaction &tran
                 rcp.paymentRequest.SerializeToString(&value);
                 vOrderForm.emplace_back("PaymentRequest", std::move(value));
             }
-            else if (!rcp.message.isEmpty()) // Message from normal xpchain:URI (xpchain:123...?message=example)
+            else if (!rcp.message.isEmpty()) // Message from normal bitcoin:URI (bitcoin:123...?message=example)
```

Comment:
Should be xpchainified, as in:
```suggestion
            else if (!rcp.message.isEmpty()) // Message from normal xpchain:URI (xpchain:123...?message=example)
```